### PR TITLE
Translation on windows fails

### PR DESCRIPTION
**Pull request by luzip665**
Bug: On german Windows exaile speaks english.

I added setting os.environ['LANG'] which helps mostly.

Only the main menu isn't translated yet.

Additionally, because it depends on windows, it solves https://github.com/exaile/exaile/issues/799

fixes: https://github.com/exaile/exaile/issues/799
fixes: https://github.com/exaile/exaile/issues/51

### CODE REVIEW

**Comment by luzip665** [comment]
On linux the manual setting of os.environ['LANGUAGE'] works.
On Win many strings are translated but some not. May depend if the string come from .ui-files?

**Comment by luzip665** [comment]
Seems to be a more general problem
https://gitlab.gnome.org/GNOME/pygobject/-/issues/422

**Comment by luzip665** [comment]
Still not translated:
* Developer tools
* Device Manager
* Equalizer
* Shortcuts
* Collection Manager
* Cover Manager

**Comment by luzip665** [comment]
Everything gets translated now.

May be there are better ways to implement the translation of the ui-xml especially for GtkTemplate. 
Hints would be appreciated.

File: xlgui/guiutil.py [hunk 1]
```diff
@@ -730,4 +732,39 @@ def without_model(tv):
             tv.scroll_to_cell(scroll_to, None, True, 0, 0)
 
 
+def get_builder(uifile: str) -> Gtk.Builder:
+    """
+    Loads Gtk.Builder with uifile.
+    In case of platform win32 each translatable node in the ui is translated here
```
Comment (sjohannes) [inline xlgui/guiutil.py:738]: This is due to the bug you mentioned (https://gitlab.gnome.org/GNOME/pygobject/-/issues/422), right? Could you add the link, so we remember why this hack is needed?

File: xl/common.py [hunk 2]
```diff
@@ -299,6 +299,27 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def platform_is_win() -> bool:
```
Comment (sjohannes) [inline xl/common.py:302]: I don't think we need these functions. On Windows `sys.platform` is always `win32` so we can just directly test for that. (And the other two functions are unused.)

File: xlgui/guiutil.py [hunk 3]
```diff
@@ -730,4 +732,39 @@ def without_model(tv):
...
+
+    :param uifile: Path to uifile
+    """
+    builder = Gtk.Builder()
+
+    if True or common.platform_is_win():
```
Comment (sjohannes) [inline xlgui/guiutil.py:744]: This is always `True`.

File: xl/common.py (hunk 2 above)
Comment (luzip665) [reply xl/common.py:302]: I would like to replace occurences of `sys.platform == 'darwin'` and `sys.platform == 'linux'.
But may be it is off topic.

File: xlgui/guiutil.py (hunk 3 above)
Comment (luzip665) [reply xlgui/guiutil.py]: oops

File: xlgui/guiutil.py (hunk 1 above)
Comment (luzip665) [reply xlgui/guiutil.py:738]: Added

**Pull request merged**